### ISSUE TITLE
Missed Django 3.0 in test matrix

### ISSUE THREAD

**Issue by Mogost** [charettes/django-polymodels#50]

**Comment by charettes** [comment]
Patches are welcome!

**Comment by kmmbvnr** [comment]
I tried to quickly add Django 3.0 compatibility. Couple of tests failed

https://github.com/kmmbvnr/django-polymodels/commit/f1cddca380c9df477c85d00ff7582ddb289b7c79


```
======================================================================
ERROR: test_delete_keep_parents (tests.test_models.BasePolymorphicModelTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/kmmbvnr/Workspace/master/!external/django-polymodels/tests/test_models.py", line 147, in test_delete_keep_parents
    self.assertEqual(snake.animal_ptr.content_type, animal_content_type)
  File "/home/kmmbvnr/Workspace/master/!external/django-polymodels/.tox/py37-master/lib/python3.7/site-packages/django/db/models/fields/related_descriptors.py", line 198, in __get__
    "%s has no %s." % (self.field.model.__name__, self.field.name)
tests.models.Snake.animal_ptr.RelatedObjectDoesNotExist: Snake has no animal_ptr.

======================================================================
FAIL: test_field_deconstruction (tests.test_fields.PolymorphicTypeFieldTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/kmmbvnr/Workspace/master/!external/django-polymodels/tests/test_fields.py", line 194, in test_field_deconstruction
    'polymorphic_type': 'polymodels.Foo',
  File "/home/kmmbvnr/Workspace/master/!external/django-polymodels/tests/test_fields.py", line 162, in assertDeconstructionEqual
    self.assertEqual(field.deconstruct(), deconstructed)
AssertionError: Tuples differ: ('foo[46 chars]], {'on_delete': <function CASCADE at 0x7fe2d7[76 chars]oo'}) != ('foo[46 chars]], {'polymorphic_type': 'polymodels.Foo', 'on_[42 chars]00>})

First differing element 3:
{'on_delete': <function CASCADE at 0x7fe2d7[75 chars]Foo'}
{'polymorphic_type': 'polymodels.Foo', 'on_[41 chars]200>}

  ('foo',
   'polymodels.fields.PolymorphicTypeField',
   [],
   {'on_delete': <function CASCADE at 0x7fe2d7a0d200>,
-   'polymorphic_type': 'polymodels.Foo',
?                                       ^

+   'polymorphic_type': 'polymodels.Foo'})
?                                       ^^

-   'to': 'contenttypes.contenttype'})

----------------------------------------------------------------------

```

**Comment by charettes** [comment]
Released 1.6.0.
